packages/playwright-core/src/server/fetch.ts

```diff
@@ -422,14 +422,35 @@ export abstract class APIRequestContext extends SdkObject {
 
         const chunks: Buffer[] = [];
         const notifyBodyFinished = () => {
+          function relativeTime(time: number | undefined): number {
+            if (!time)
+              return -1;
+            return time - startAt;
+          }
+          const endAt = monotonicTime();
+          // spec: https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming
+          const timing: channels.ResourceTiming = {
+            startTime: startAt,
```

Comment:
The docs say `startTime` is a wall time, not monotonic time.

Reply:
done in https://github.com/microsoft/playwright/pull/32647/commits/b717257013f459955f0172483e1b52b55ac7305d